package.json

```diff
@@ -3,18 +3,24 @@
     "version": "1.1.0",
     "description": "bitcoin SPV proofs in Solidity",
     "scripts": {
-        "compile": "node compile.js",
-        "test": "mocha --timeout 5000"
+        "coverage": "npx solidity-coverage",
```

Comment:
`npm run coverage` failed, with errors below (all tests failed). When I executed `truffle test` all tests passed.

```
  1) Contract: BTCUtils
       gets the last bytes correctly:
     Error: Returned error: VM Exception while processing transaction: revert
      at Object.ErrorResponse (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-helpers/src/errors.js:29:1)
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-requestmanager/src/index.js:140:1
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/packages/truffle-provider/wrapper.js:112:1
      at XMLHttpRequest.request.onreadystatechange (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3/~/web3-providers-http/src/index.js:96:1)
      at XMLHttpRequestEventTarget.dispatchEvent (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
      at XMLHttpRequest._setReadyState (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:208:1)
      at XMLHttpRequest._onHttpResponseEnd (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:318:1)
      at IncomingMessage.<anonymous> (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:289:47)
      at endReadableNT (_stream_readable.js:1139:12)
      at processTicksAndRejections (internal/process/task_queues.js:81:17)

  2) Contract: BTCUtils
       errors if slice is larger than the bytearray:
     AssertionError: expected 'Returned error: VM Exception while processing transaction: revert' to include 'Underflow during subtraction.'
      at Context.it (test/BTCUtils.test.js:42:14)

  3) Contract: BTCUtils
       reverses endianness:
     Error: Returned error: VM Exception while processing transaction: revert
      at Object.ErrorResponse (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-helpers/src/errors.js:29:1)
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-requestmanager/src/index.js:140:1
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/packages/truffle-provider/wrapper.js:112:1
      at XMLHttpRequest.request.onreadystatechange (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3/~/web3-providers-http/src/index.js:96:1)
      at XMLHttpRequestEventTarget.dispatchEvent (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
      at XMLHttpRequest._setReadyState (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:208:1)
      at XMLHttpRequest._onHttpResponseEnd (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:318:1)
      at IncomingMessage.<anonymous> (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:289:47)
      at endReadableNT (_stream_readable.js:1139:12)
      at processTicksAndRejections (internal/process/task_queues.js:81:17)

  4) Contract: BTCUtils
       converts big-endian bytes to integers:
     Error: Returned error: VM Exception while processing transaction: revert
      at Object.ErrorResponse (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-helpers/src/errors.js:29:1)
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-requestmanager/src/index.js:140:1
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/packages/truffle-provider/wrapper.js:112:1
      at XMLHttpRequest.request.onreadystatechange (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3/~/web3-providers-http/src/index.js:96:1)
      at XMLHttpRequestEventTarget.dispatchEvent (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
      at XMLHttpRequest._setReadyState (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:208:1)
      at XMLHttpRequest._onHttpResponseEnd (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:318:1)
      at IncomingMessage.<anonymous> (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:289:47)
      at endReadableNT (_stream_readable.js:1139:12)
      at processTicksAndRejections (internal/process/task_queues.js:81:17)

  5) Contract: BTCUtils
       implements bitcoin's hash160:
     Error: Returned error: VM Exception while processing transaction: revert
      at Object.ErrorResponse (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-helpers/src/errors.js:29:1)
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-requestmanager/src/index.js:140:1
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/packages/truffle-provider/wrapper.js:112:1
      at XMLHttpRequest.request.onreadystatechange (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3/~/web3-providers-http/src/index.js:96:1)
      at XMLHttpRequestEventTarget.dispatchEvent (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
      at XMLHttpRequest._setReadyState (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:208:1)
      at XMLHttpRequest._onHttpResponseEnd (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:318:1)
      at IncomingMessage.<anonymous> (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:289:47)
      at endReadableNT (_stream_readable.js:1139:12)
      at processTicksAndRejections (internal/process/task_queues.js:81:17)

  6) Contract: BTCUtils
       implements bitcoin's hash256:
     Error: Returned error: VM Exception while processing transaction: revert
      at Object.ErrorResponse (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-helpers/src/errors.js:29:1)
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-requestmanager/src/index.js:140:1
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/packages/truffle-provider/wrapper.js:112:1
      at XMLHttpRequest.request.onreadystatechange (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3/~/web3-providers-http/src/index.js:96:1)
      at XMLHttpRequestEventTarget.dispatchEvent (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
      at XMLHttpRequest._setReadyState (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:208:1)
      at XMLHttpRequest._onHttpResponseEnd (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:318:1)
      at IncomingMessage.<anonymous> (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:289:47)
      at endReadableNT (_stream_readable.js:1139:12)
      at processTicksAndRejections (internal/process/task_queues.js:81:17)

  7) Contract: BTCUtils
       extracts a sequence from a witness input as LE and int:
     Error: Returned error: VM Exception while processing transaction: revert
      at Object.ErrorResponse (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-helpers/src/errors.js:29:1)
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-requestmanager/src/index.js:140:1
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/packages/truffle-provider/wrapper.js:112:1
      at XMLHttpRequest.request.onreadystatechange (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3/~/web3-providers-http/src/index.js:96:1)
      at XMLHttpRequestEventTarget.dispatchEvent (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
      at XMLHttpRequest._setReadyState (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:208:1)
      at XMLHttpRequest._onHttpResponseEnd (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:318:1)
      at IncomingMessage.<anonymous> (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:289:47)
      at endReadableNT (_stream_readable.js:1139:12)
      at processTicksAndRejections (internal/process/task_queues.js:81:17)

  8) Contract: BTCUtils
       extracts a sequence from a legacy input as LE and int:
     Error: Returned error: VM Exception while processing transaction: revert
      at Object.ErrorResponse (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-helpers/src/errors.js:29:1)
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-requestmanager/src/index.js:140:1
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/packages/truffle-provider/wrapper.js:112:1
      at XMLHttpRequest.request.onreadystatechange (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3/~/web3-providers-http/src/index.js:96:1)
      at XMLHttpRequestEventTarget.dispatchEvent (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
      at XMLHttpRequest._setReadyState (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:208:1)
      at XMLHttpRequest._onHttpResponseEnd (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:318:1)
      at IncomingMessage.<anonymous> (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:289:47)
      at endReadableNT (_stream_readable.js:1139:12)
      at processTicksAndRejections (internal/process/task_queues.js:81:17)

  9) Contract: BTCUtils
       extracts an outpoint as bytes:
     Error: Returned error: VM Exception while processing transaction: revert
      at Object.ErrorResponse (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-helpers/src/errors.js:29:1)
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3-eth/~/web3-core-requestmanager/src/index.js:140:1
      at /Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/packages/truffle-provider/wrapper.js:112:1
      at XMLHttpRequest.request.onreadystatechange (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/web3/~/web3-providers-http/src/index.js:96:1)
      at XMLHttpRequestEventTarget.dispatchEvent (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request-event-target.js:34:1)
      at XMLHttpRequest._setReadyState (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:208:1)
      at XMLHttpRequest._onHttpResponseEnd (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:318:1)
      at IncomingMessage.<anonymous> (/Users/jakub/workspace/bitcoin-spv/node_modules/truffle/build/webpack:/~/xhr2-cookies/dist/xml-http-request.js:289:47)
      at endReadableNT (_stream_readable.js:1139:12)
      at processTicksAndRejections (internal/process/task_queues.js:81:17)

Cleaning up...
Shutting down testrpc-sc (pid 28494)
Event trace could not be read.
Error: ENOENT: no such file or directory, open './allFiredEvents'
Exiting without generating coverage...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bitcoin-spv@1.1.0 coverage: `npx solidity-coverage`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the bitcoin-spv@1.1.0 coverage script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jakub/.npm/_logs/2019-07-30T10_27_12_558Z-debug.log
```

Reply:
My local env fell afoul of this bug, might be related:
https://github.com/sc-forks/solidity-coverage/issues/234

Reply:
well, I shouldn't say my local env. "a change I made to the repo" :roll_eyes:

Reply:
Can you pull latest and try again?

Reply:
It worked now 🎉